### DISCUSSION TITLE
Filter folders based on user

### DISCUSSION THREAD

**Discussion by dmarcos89** [Studio-42/elFinder#3334] [Q&A]
Hi have a multi-tenancy app, and would like to show/hide folders based on the company of the logged user.
What's the best way to achieve something like this?
Would appreciate any suggestion or guidance :) 
Thanks!

**Comment by qdequippe** [comment]
@dmarcos89 I use the Symfony Bundle but you can generate the connector based on your logged user (see https://github.com/Studio-42/elFinder/wiki/Connector-configuration-options#root-options).

```
$opts = array(
    'locale' => '',
    'roots'  => array(
        array(
            'driver' => 'LocalFileSystem',
            'path'   => '/path/to/files/', <--- Change the path based on logged user
            'URL'    => 'http://localhost/to/files/' <--- or the URL...
        )
    )
);

// run elFinder
$connector = new elFinderConnector(new elFinder($opts));
$connector->run();
```
